Source/WebKit/Scripts/copy-webcontent-resources-to-private-headers.sh

```diff
@@ -39,7 +39,9 @@ PROCESSED_INFOPLIST="${BUILT_PRODUCTS_DIR}/${INFOPLIST_PATH}"
 UNPROCESSED_INFOPLIST="${INFOPLIST_FILE}"
 COPIED_INFOPLIST="${WEB_CONTENT_RESOURCES_PATH}/Info-WebContent.plist"
 echo "Copying Info.plist from ${UNPROCESSED_INFOPLIST} to ${COPIED_INFOPLIST}"
-ditto "${UNPROCESSED_INFOPLIST}" "${COPIED_INFOPLIST}"
+if [ -e "${UNPROCESSED_INFOPLIST}" ]; then
+    ditto "${UNPROCESSED_INFOPLIST}" "${COPIED_INFOPLIST}"
+fi
```

Comment:
Not having `UNPROCESSED_INFOPLIST` exist in some configurations may raise workspace validation warnings, since the plist is listed as an input dependency of this script phase. I don't think you should try and fix it now, but if this comes up in the future (especially as we update the iOS SDK on various build bots), we might need to switch this script phase to emit discovered dependencies.